xmpp/auth.py

```diff
@@ -37,10 +37,12 @@ def H(some):
     if IS_PYTHON3 and isinstance(some, str):
         some = some.encode("utf-8")
     return md5(some).digest()
+
```

Comment:
Keep the whitespace consistent between HH&H and H&C (either both have whitespace, or neither have whitespace).

Bonus points for removing the `print("HH:some:", some)` from HH, that looks like left over debugging from #45.

Reply:
Hi. I've just addressed this with e6f4667.